pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -147,6 +147,9 @@ abstract contract ManagedPoolSettings is NewBasePool, ProtocolFeeCache, IManaged
 
         // If true, only addresses on the manager-controlled allowlist may join the pool.
         _setMustAllowlistLPs(params.mustAllowlistLPs);
+
+        // Joins and exits are enabled by default on start.
+        _setJoinExitEnabled(true);
```

Comment:
I guess we don't want to change the current behavior with this feature, do we?

Reply:
Hmm, looks like we don't apply the check on initialisation atm.

Reply:
Probably ok? I guess it depends on the intent of this feature, and why Integrations needed it. We do check the allowList on initialization, which is sort of similar to checking this flag, so I could see it either way.

Reply:
I don't see a reason to have them be disabled at start - the reason why we need to disable them is very specific. If needed, it can be done atomically by the deployer, and it'd complicate the constructor to add this.

It does feel a bit weird to not provide the toggle though.